config/core/build-configs-chromeos.yaml

```diff
@@ -29,6 +29,7 @@ chromeos_variants: &chromeos-variants
           - 'cros://chromeos-5.10/x86_64/chromeos-amd-stoneyridge.flavour.config+x86-chromebook'
           - 'cros://chromeos-5.10/x86_64/chromeos-intel-denverton.flavour.config+x86-chromebook'
           - 'cros://chromeos-5.10/x86_64/chromeos-intel-pineview.flavour.config+x86-chromebook'
+          - 'cros://chromeos-5.10/x86_64/chromiumos-x86_64.flavour.config+x86-chromebook'
```

Comment:
Do we actually need the `x86-chromebook` framgment for QEMU jobs?

Reply:
It is extracted from SDK configs according to platform/board configuration.
Looking inside, i can see one critical entry CONFIG_DRM_VIRTIO_GPU (but nothing else). 
Other flavours contain it too, so theoretically we can use pineview or stoneyridge for QEMU, but i am not sure how critical is other options (there is plenty for SND and etc).

Reply:
Yes, definitely and exactly as you mentioned, because default ChromeOS fragments doesnt include serial ports for example. And we are using QEMU serial port for interfacing with DUT.

Reply:
Yes